pkg/pullsync/pullsync.go

```diff
@@ -65,30 +66,34 @@ type Interface interface {
 }
 
 type Syncer struct {
-	streamer   p2p.Streamer
-	metrics    metrics
-	logger     log.Logger
-	storage    pullstorage.Storer
-	quit       chan struct{}
-	wg         sync.WaitGroup
-	unwrap     func(swarm.Chunk)
-	validStamp postage.ValidStampFn
+	streamer       p2p.Streamer
+	metrics        metrics
+	logger         log.Logger
+	storage        pullstorage.Storer
+	quit           chan struct{}
+	wg             sync.WaitGroup
+	unwrap         func(swarm.Chunk)
+	validStamp     postage.ValidStampFn
+	radius         postage.RadiusChecker
+	overlayAddress common.Address
 
 	Interface
 	io.Closer
 }
 
-func New(streamer p2p.Streamer, storage pullstorage.Storer, unwrap func(swarm.Chunk), validStamp postage.ValidStampFn, logger log.Logger) *Syncer {
+func New(streamer p2p.Streamer, storage pullstorage.Storer, unwrap func(swarm.Chunk), validStamp postage.ValidStampFn, logger log.Logger, radius postage.RadiusChecker, overlayAddress common.Address) *Syncer {
```

Comment:
should be swarm.Address

Reply:
https://github.com/ethersphere/bee/blob/e0171d2588c8a1d4c824b4f496874c966bbdc2d7/pkg/node/node.go#L294

All of the initialization functions are using common.Address. There is little difference. One way is to declare it again in `node.go` but that would be redundant.